### PR TITLE
Notebook suggestions

### PR DESCRIPTION
**Pull request by martinfleis**
Hey, some suggestions on the notebook.

### CODE REVIEW

File: docs/user_guide/preprocessing/roundabout_simplification.ipynb [hunk 1]
```diff
@@ -4,11 +4,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Road Network Simplification of Roundabouts\n",
```
Comment (martinfleis) [inline docs/user_guide/preprocessing/roundabout_simplification.ipynb:7]: I made the title shorted and removed capitalisation of the words.

File: docs/user_guide/preprocessing/roundabout_simplification.ipynb [hunk 2]
```diff
@@ -4,11 +4,13 @@
...
-    "The following code exemplifies a simple process for road network simplification of roundabouts using [momepy.roundabout_simplification()](http://docs.momepy.org/en/latest/generated/momepy.roundabout_simplification.html?highlight=momepy.roundabout_simplification).\n",
+    "# Simplification of roundabouts\n",
     "\n",
-    "For this example we will fetch some data from [OpenStreetMap](https://www.openstreetmap.org/#map=6/40.007/-2.488) (using `osmnx`). However, most other data sources should also work as long as their topology has been corrected and can be stored in a GeoDataFrame.\n",
-    "The code below uses only geometry operations to perform the simplification."
+    "Analysis using street network can be sensitive to the actual geometry representation. For example, one roundabout can be typically represented as 4 or more nodes, while it is topologically only one. Hence measurements attempting to capture topology (e.g. centrality) or node density will be skewed.\n",
```
Comment (martinfleis) [inline docs/user_guide/preprocessing/roundabout_simplification.ipynb:9]: Added a bit of context for the intro.

File: docs/user_guide/preprocessing/roundabout_simplification.ipynb [hunk 3]
```diff
@@ -247,8 +249,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Playing with the different parameters\n",
-    "### _default_"
+    "## Default behavior"
```
Comment (martinfleis) [inline docs/user_guide/preprocessing/roundabout_simplification.ipynb:252]: Simplifying structure

File: docs/user_guide/preprocessing/roundabout_simplification.ipynb [hunk 4]
```diff
@@ -269,13 +270,6 @@
     "edges_output = mm.roundabout_simplification(edges)"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "*_ShapelyDeprecationWarning should be resolved in further versions_."
-   ]
-  },
```
Comment (martinfleis) [inline docs/user_guide/preprocessing/roundabout_simplification.ipynb:278]: Warnings are not shown in the docs.

File: docs/user_guide/preprocessing/roundabout_simplification.ipynb [hunk 5]
```diff
@@ -305,7 +299,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Using the output column _```simplification_group```_ we can count the number of roundabouts that were simplified"
+    "Using the output column `'simplification_group'` we can count the number of roundabouts that were simplified:"
```
Comment (martinfleis) [inline docs/user_guide/preprocessing/roundabout_simplification.ipynb:302]: Removing unnecessary formatting.

File: docs/user_guide/preprocessing/roundabout_simplification.ipynb [hunk 6]
```diff
@@ -379,7 +373,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It's worth investigating why some roundabouts are still missing from simplification. Since the selection of roundabouts is based on the resulting polygons after polygonizing the road network let's use the [Circular Compactness](https://docs.momepy.org/en/stable/generated/momepy.CircularCompactness.html) parameter of the resulting polygons as the main selection attribute."
+    "It's worth investigating why some roundabouts are still missing from simplification. Since the selection of roundabouts is based on the resulting polygons after polygonizing the road network let's use the circular compactness (measurable as `momepy.CircularCompactness`) parameter of the resulting polygons as the main selection attribute."
```
Comment (martinfleis) [inline docs/user_guide/preprocessing/roundabout_simplification.ipynb:376]: We have different versions of the docs, we cannot hard-code the link to one of those.

File: docs/user_guide/preprocessing/roundabout_simplification.ipynb [hunk 7]
```diff
@@ -498,11 +492,11 @@
...
     "The above helps to exemplify two main reasons why some roundabouts are not being selected:\n",
     "1. They are crossed by other roads; which breaks them into multiple polygons\n",
-    "    - eg. `index`: ```[314, 312, 304]```\n",
+    "    - eg. `index`: `[314, 312, 304]`\n",
     "1. They don't meet the `circom_threshold`\n",
-    "     - eg. `index`: ```[162]``` --> (0.676) \n",
+    "     - eg. `index`: `[162]` --> (0.676) \n",
     "     \n",
-    "## Lowering _**circom_threshold**_ & **include_adjacent**_ = False\n",
+    "## Adapting parameters\n",
```
Comment (martinfleis) [inline docs/user_guide/preprocessing/roundabout_simplification.ipynb:499]: This subtitle looks bad when rendered in docs. Simplified.

**Review by gregmaya — APPROVED** [review]
